interfaces/builtin/desktop.go

```diff
@@ -491,6 +491,51 @@ dbus (send)
 /etc/xdg/user-dirs.conf r,
 /etc/xdg/user-dirs.defaults r,
 /run/udev/tags/seat{,/**} r,
+
+# KDE Plasma specific extension
+
+# Used by the KCrash handler
+@{PROC}/sys/kernel/core_pattern r,
+
+# So that KSplash disappears when appropriate
+dbus (receive, send)
+    bus=session
+    path=/KSplash
+    interface=org.kde.KSplash
+    member=setStage
+    peer=(label=unconfined),
+dbus (receive, send)
+    bus=session
+    path=/KSplash
+    interface=org.freedesktop.DBus.Introspectable
+    member=Introspect
+    peer=(label=unconfined),
+
+dbus (receive, send)
+    bus=session
+    path=/KSMServer
+    interface=org.kde.KSMServerInterface
+    member=restoreSession
+    peer=(label=unconfined),
+dbus (receive, send)
+    bus=session
+    path=/KSMServer
+    interface=org.freedesktop.DBus.Introspectable
+    member=Introspect
+    peer=(label=unconfined),
```

Comment:
Similarly, these seem to relate to a daemon running as `snap.plasma-desktop-session.plasma-ksmserver`, so `label=unconfined` seems like it won't match any messages. If nothing outside the snap needs to talk to ksmserver, we might not need to mention it at all.